ui_vue/cypress/e2e/signUp/signup.cy.ts

```diff
@@ -0,0 +1,38 @@
+// Import dom data
+import domData from '../../lib/signIn_lib/signIn.dom.data';
+
+// Import configData
+import configData from '../../config';
+
+// Import constants
+import constants from '../../cypress.constants';
+
+
+// Cypress testing for signin with local credentials
+describe('Sign Up', () => {
+
+  it('passwords dont match', () => {
```

Comment:
don't

Reply:
changed it to 'passwords don\'t match'.

Cannot do it like "passwords don't match" because of the single quote eslint rule.